composer.json

```diff
@@ -11,19 +11,19 @@
         }
     ],
     "require": {
-        "php": ">=7.1",
-        "swarrot/swarrot": "^2.1.1|^3.0",
-        "psr/log": "~1.0",
-        "symfony/framework-bundle": "~2.4|~3.0|~4.0",
-        "symfony/console": "~2.4|~3.0|~4.0",
-        "symfony/yaml": "~2.4|~3.0|~4.0"
+        "php": ">=7.2",
+        "swarrot/swarrot": "^3.3",
+        "psr/log": "~1.1",
+        "symfony/framework-bundle": "~4.2",
```

Comment:
We should actually have direct requirements on the DI and Config components instead of relying on transitive deps of FrameworkBundle, as these are the actual components for which you drop BC.

Btw, I don't even think we depend on FrameworkBundle (we do depend on HttpKernel though)

Reply:
Done